JustSaying.Sample.Restaurant.KitchenConsole/OrderPlacedEventHandler.cs

```diff
@@ -0,0 +1,37 @@
+using JustSaying.Messaging;
+using JustSaying.Messaging.MessageHandling;
+using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
```

Comment:
Nit: sort using statements where System directives take precedence over the others. I guess we should find a way to editor-config that as the default.

Reply:
done